### ISSUE TITLE
'az network dns zone -h' has incorrect text

### ISSUE THREAD

**Issue by jtuliani** [Azure/azure-cli#1121]
'az network dns zone -h' shows the following:

```
create: Create or update a virtual machine.
delete: Removes a DNS zone from a resource group.
export
import
list
show  : Gets a DNS zone.
update
```

Several problems with this:
- 'Create' should talk about DNS zones, not VMs
- No text for export, import, list or update
- 'Show': text isn't useful. Replace with "Displays the properties of the DNS zone. Does not list DNS records (see 'az network dns record-set list')."

**Comment by JasonRShaver** [comment]
* fix help

**Comment by jtuliani** [comment]
We can't fix the help purely through Swagger descriptions: Swagger doesn't support import/export; nor zone PATCH (for 'update'), and for List there are two methods in Swagger (so not clear which to use).

Need to provide help text at CLI level.  Proposed text below.

Create: Creates or updates DNS zone properties.  Does not modify DNS records within the zone.
Delete: Deletes a DNS zone. WARNING: All DNS records in the zone will also be deleted. This operation cannot be undone.
Export: Export a DNS zone as a DNS zone file.
Import: Import a DNS zone from a DNS zone file.
List: List DNS zones in a resource group or subscription.
Show: Gets DNS zone parameters.  Does not show DNS records within the zone.
Update: Updates DNS zone properties.  Does not modify DNS records within the zone.

**Comment by troydai** [comment]
Fixed in e470852
